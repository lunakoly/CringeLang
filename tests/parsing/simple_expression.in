5 + 10 - "test" - -12 + %010 -#eff * 20 * 30