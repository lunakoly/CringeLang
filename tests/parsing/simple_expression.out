==== Raw AST ====
[[(((((5 + 10) - "test") - (-12)) + 2) - ((3839 * 20) * 30))]]

==== Resolved AST ====
[[(((((5 + 10) - "test") - (-12)) + 2) - ((3839 * 20) * 30))]]

==== Diagnostics ====

==== Done ====
